### ISSUE TITLE
unwanted loop

### ISSUE THREAD

**Issue by HUBBA-ER** [HUBBA-ER/ChooseACave#1]
i was doing initial testing for the 'info' command and it worked... until you finished it would go back to menu then info then menu in an endless loop. lets fix this before testing 'credits' then we can code initial game

**Comment by tripplyons** [comment]
I have a working version that has not been uploading. I will try manually uploading it now.

**Comment by HUBBA-ER** [comment]
okay but we should re-initialize the 'credits' function

**Comment by tripplyons** [comment]
Okay

**Comment by HUBBA-ER** [comment]
lets work on that now

**Comment by tripplyons** [comment]
Okay

**Comment by MeowCatPanda** [comment]
can i help??

**Comment by HUBBA-ER** [comment]
yes please check through the new version for bugs on codingground
